src/dbSta/src/dbNetwork.cc

```diff
@@ -4190,6 +4239,18 @@ Net* dbNetwork::getFlatNet(Net* net) const
   return dbToSta(db_net);
 }
 
+dbModInst* dbNetwork::getModInst(Instance* inst) const
+{
+  if (!inst) {
+    return nullptr;
+  }
+
+  dbInst* db_inst = nullptr;
+  dbModInst* db_mod_inst = nullptr;
+  staToDb(inst, db_inst, db_mod_inst);
+  return db_mod_inst;
```

Comment:
What is this is a db_inst?  It nullptr an expected return value in that case?  This seems similar to the other APIs I added asserts to.